test/NuGet.Core.Tests/NuGet.Protocol.Tests/HttpSource/HttpSourceAuthenticationHandlerTests.cs

```diff
@@ -19,7 +20,7 @@ public class HttpSourceAuthenticationHandlerTests
         [Fact]
         public void Constructor_WithSourceCredentials_InitializesClientHandler()
         {
-            var packageSource = new PackageSource("http://package.source.net", "source")
+            var packageSource = new PackageSource("http://package.source.test", "source")
```

Comment:
Why did this matter? :D

Reply:
Because Damon said so.

Also, it ensures that if someone refactors some code, or copies the test and makes some changes that causes real HTTP requests to be sent out, we don't start making requests to a hostname not owned by Microsoft.